docs/content/developers/design-keuzes.md

```diff
@@ -293,3 +293,77 @@ en [Facebook](https://developers.facebook.com/docs/graph-api/reference/photo/#Cr
 Verschillende bronnen zijn hier wel over verdeeld, zoals
 [REST API tutorial](https://restfulapi.net/resource-naming/) en [Wikipedia](https://en.wikipedia.org/wiki/Representational_state_transfer#Relationship_between_URL_and_HTTP_methods)
 maar er is gekozen om te kijken naar de praktijk en DSO.
+
+## Zaak afsluiten
+
+Een zaak wordt afgesloten door een eindstatus toe te kennen aan een `ZAAK`.
+Elk `ZAAKTYPE` moet (minimaal) 1 eindstatus kennen. De eindstatus binnen een
```

Comment:
Gezien de kardinaliteiten in ImZTC moet dit m.i. zijn: "Elk `ZAAKTYPE` heeft minimaal één 'STATUSTYPE'."